config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -55,6 +55,9 @@ spec:
          # Default nic config template for a EDPM compute node
          # These vars are edpm_network_config role vars
          edpm_network_config_hide_sensitive_logs: false
+         ctlplane_dns_nameservers:
+         - 192.168.122.80
+         dns_search_domains: []
```

Comment:
The variable is `edpm_dns_search_domains` now:
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_network_config/defaults/main.yml#L53

But this is just setting the default value anyway, so we can just leave it out of the sample file to avoid unnecessary clutter.